NEWS

```diff
@@ -2,6 +2,9 @@
 
 New in 8.3 version:
 
+* bin/postgresql-setup.in: Print warning about possible glibc collation on
```

Comment:
`s|bin/postgresql-setup.in: Print |postgresql-setup prints|` as this isn't supposed to have ChangeLog syntax

Reply:
Fixed as well.